upload/system/library/emshelper.php

```diff
@@ -657,6 +657,22 @@ public static function ipIsEnabled($ipList)
         return true;
     }
 
+    /**
+     * @param $countryList
+     * @return bool
+     */
+    public static function CountryValidator($countryList, $billingAddress)
```

Comment:
spell function name from the first small letter: like country Validator

Reply:
Done